### ISSUE TITLE
Avoid use Popen()?

### ISSUE THREAD

**Issue by Jack47** [Augusto111/STtech_test#3]
Could we avoid use `Popen()` to find the permission of files? Maybe python provides methods to `directly` traverse and get permission of files under directory? This is a improvement.

**Comment by Augusto111** [comment]
Yes, python provides the method `access()` to check if a user is authorized to read, write or execute.
In this way, we can avoid the possible manipulation between checking and splitting. And we also saved time ,which we no longer used the shell to find the permission of files.
